csharp/ql/src/semmle/code/csharp/Type.qll

```diff
@@ -999,5 +999,63 @@ class TypeMention extends @type_mention {
 @type_or_ref getTypeRef(@type type) {
```

Comment:
We should probably cache this predicate now that it relies on some non-trivial calculations.